docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-title: Desktop Access with Active Directory (Manual)
+title: Register Windows Desktops with Active Directory (Manual)
```

Comment:
This guide isn't really about registering desktops. How about "Connecting to desktops" instead?